agent/bench-scripts/pbench-user-benchmark

```diff
@@ -160,7 +160,7 @@ function record_iteration {
 }
 record_iteration ${iteration} ${iteration_name} ${benchmark_bin}
 # on abnormal exit, make sure that the metadata log exists and is complete.
-trap "pbench-metadata-log --group=$tool_group --dir=$benchmark_run_dir int" INT QUIT
+trap "pbench-metadata-log --group=$tool_group --dir=$benchmark_run_dir int; benchmark_iterations" INT QUIT
```

Comment:
So this `trap` appears to do two things:

 1. Runs the command `pbench-metadata-log --group=$tool_group --dir=$benchmark_run_dir int`
 1. Tries to run `benchmark_iterations`, which does not appear to be a command or a variable

It seems that it would be useful to just move `$benchmark_iterations` from `$pbench_tmp` to `$pbench_run` so that it is preserved in the data set.  It would be useful for debugging.